modules/test/playwright/tests/object-web/utils/mockObjectFieldsObjectEntry.ts

```diff
@@ -0,0 +1,362 @@
+			objectFieldSettings: [],
+			readOnly: 'false',
+			readOnlyConditionExpression: '',
+			required: false,
+			state: false,
+			system: false,
+			type: 'String',
+			unique: false,
+		},
+		{
+			DBType: 'String',
+			businessType: 'Picklist',
+			indexed: true,
+			indexedAsKeyword: false,
+			indexedLanguageId: 'en_US',
+			label: {
+				en_US: objectFieldBusinessTypesInfo.picklist.label,
+			},
+			listTypeDefinitionExternalReferenceCode:
+				listTypeDefinition.externalReferenceCode,
+			localized: false,
+			name: objectFieldBusinessTypesInfo.picklist.name,
+			objectFieldSettings: [],
+			readOnly: 'false',
+			readOnlyConditionExpression: '',
+			required: false,
+			state: false,
+			system: false,
+			type: 'String',
+			unique: false,
+		},
+		{
+			DBType: 'BigDecimal',
+			businessType: 'PrecisionDecimal',
+			indexed: true,
+			indexedAsKeyword: false,
+			indexedLanguageId: '',
+			label: {
+				en_US: objectFieldBusinessTypesInfo.precisionDecimal.label,
+			},
+			localized: false,
+			name: objectFieldBusinessTypesInfo.precisionDecimal.name,
+			objectFieldSettings: [],
+			readOnly: 'false',
+			readOnlyConditionExpression: '',
+			required: false,
+			state: false,
+			system: false,
+			type: 'BigDecimal',
+			unique: false,
+		},
+		{
+			DBType: 'Clob',
+			businessType: 'RichText',
+			indexed: true,
+			indexedAsKeyword: false,
+			indexedLanguageId: 'en_US',
+			label: {
+				en_US: objectFieldBusinessTypesInfo.richText.label,
+			},
+			localized: false,
+			name: objectFieldBusinessTypesInfo.richText.name,
+			objectFieldSettings: [],
+			readOnly: 'false',
+			readOnlyConditionExpression: '',
+			required: false,
+			state: false,
+			system: false,
+			type: 'Clob',
+			unique: false,
+		},
+		{
+			DBType: 'String',
+			businessType: 'Text',
+			indexed: true,
+			indexedAsKeyword: false,
+			indexedLanguageId: 'en_US',
+			label: {
+				en_US: objectFieldBusinessTypesInfo.text.label,
+			},
+			localized: false,
+			name: objectFieldBusinessTypesInfo.text.name,
+			objectFieldSettings: [],
+			readOnly: 'false',
+			readOnlyConditionExpression: '',
+			required: false,
+			state: false,
+			system: false,
+			type: 'String',
+			unique: false,
+		},
+	];
+}
+
+export function mockObjectFieldsObjectEntry(
+	listTypeDefinition: ListTypeDefinition,
+	listTypeDefinitionItems: string[]
+) {
+	const objectFieldBusinessTypeLabelName =
+		{} as objectFieldBusinessTypesLabelName;
+
+	const setLabelName = (
+		target: objectFieldBusinessTypesLabelName,
+		businessType: string,
+		{label, name}
+	) => {
+		Object.defineProperty(target, businessType, {value: {label, name}});
+	};
+
+	objectFieldBusinessTypes.forEach((objectFieldBusinessType) => {
+		setLabelName(
+			objectFieldBusinessTypeLabelName,
+			objectFieldBusinessType,
+			{
+				label: `${objectFieldBusinessType}ObjectFieldLabel`,
+				name: `${objectFieldBusinessType}ObjectFieldName`,
+			}
+		);
+	});
+
+	return {
+		objectEntry: mockObjectEntry(
```

Comment:
Hey @igor-franca, shouldn't be objectEntries?

Reply:
Hey Dan, at this moment the function is only returning one entry.